src/components/Tabs/Tabs.tsx

```diff
@@ -19,40 +30,38 @@ const Tabs: FC<TabsProps> = ({
   variant = TabVariants.underline,
   isExpanded = false,
 }) => (
-  <BaseTabsWrapper
-    $isExpanded={isExpanded}
-    $variant={variant}
-    className={CLX_COMPONENT}
-    paddingSize={
-      variant === TabVariants.segmented ? SpaceSizes.xs : SpaceSizes.none
-    }
+  <Provider
+    value={{
+      selectedPatternMatcher,
+      selectedValue,
+      onSelectTab,
```

Comment:
```suggestion
      selectTab: onSelectTab,
```
I think we can change the `onSelectTab` property name to `selectTab` because we are not using it in the `Tab` component as a handler but a method to select tab so the `on` prefix is unnecessary